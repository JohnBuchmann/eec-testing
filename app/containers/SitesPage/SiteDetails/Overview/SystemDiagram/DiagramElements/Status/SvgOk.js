/**
 * Ok icon
 */

import React from 'react';
import { Colors } from 'Theme';

/**
 * Display the core of ok icon for svg
 */
function SvgOk() {
  return (
    <>
      <desc>Ok icon core</desc>
      <defs>
        <filter id="3zjp5py6ka">
          <feColorMatrix
            in="SourceGraphic"
            values="0 0 0 0 1.000000 0 0 0 0 1.000000 0 0 0 0 1.000000 0 0 0 1.000000 0"
          />
        </filter>
      </defs>
      <g fill="none" fillRule="evenodd">
        <g>
          <g>
            <g>
              <g
                filter="url(#3zjp5py6ka)"
                transform="translate(-876 -284) translate(98 178) translate(774 102) translate(4 4)"
              >
                <g>
                  <path
                    fill={Colors.black}
                    fillRule="nonzero"
                    d="M16.225 0L18 1.731 6.144 15 0 8.983 1.661 7.133 6.028 11.41z"
                    transform="translate(3 5)"
                  />
                </g>
              </g>
            </g>
          </g>
        </g>
      </g>
    </>
  );
}

export default SvgOk;
