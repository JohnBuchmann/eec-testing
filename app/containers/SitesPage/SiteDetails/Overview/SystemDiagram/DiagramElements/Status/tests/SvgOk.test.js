import { shallow } from 'enzyme';
import React from 'react';
import SvgOk from '../SvgOk';

describe('SvgOk', () => {
  it('should render SVG core structure', () => {
    const actual = shallow(<SvgOk />);

    expect(actual).toMatchSnapshot();
  });
});
