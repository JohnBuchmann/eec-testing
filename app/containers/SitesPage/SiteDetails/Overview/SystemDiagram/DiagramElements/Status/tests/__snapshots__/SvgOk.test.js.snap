// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgOk should render SVG core structure 1`] = `
<Fragment>
  <desc>
    Ok icon core
  </desc>
  <defs>
    <filter
      id="3zjp5py6ka"
    >
      <feColorMatrix
        in="SourceGraphic"
        values="0 0 0 0 1.000000 0 0 0 0 1.000000 0 0 0 0 1.000000 0 0 0 1.000000 0"
      />
    </filter>
  </defs>
  <g
    fill="none"
    fillRule="evenodd"
  >
    <g>
      <g>
        <g>
          <g
            filter="url(#3zjp5py6ka)"
            transform="translate(-876 -284) translate(98 178) translate(774 102) translate(4 4)"
          >
            <g>
              <path
                d="M16.225 0L18 1.731 6.144 15 0 8.983 1.661 7.133 6.028 11.41z"
                fill="#000000"
                fillRule="nonzero"
                transform="translate(3 5)"
              />
            </g>
          </g>
        </g>
      </g>
    </g>
  </g>
</Fragment>
`;
